addons/im_livechat/static/src/embed/core/livechat_service.js

```diff
@@ -173,16 +167,17 @@ export class LivechatService {
      */
     async sendFeedback(uuid, rate, reason) {
         await this.rpc("/im_livechat/feedback", { reason, rate, uuid });
+        const emojiSrc = `${session.origin}/rating/static/src/img/rating_${
+            RATING[Object.keys(RATING).find((key) => RATING[key] === rate)]
+        }.png`;
         await this.rpc("/im_livechat/chat_post", {
             uuid,
-            message_content: sprintf(_t("Rating: %s"), RATING_TO_EMOJI[rate]),
+            message_content: sprintf(
+                _t("Rating: %s"),
+                `<img height="25" width="25" src="${emojiSrc}" alt="rating_${rate}" /> ${reason}`
+            ),
```

Comment:
This is not the first occurrence (and not necessarily blocking), but it would be better if that was meta-data rather than hard-coded value.

I'm thinking maybe we could have either extra table(s)/field(s) for message meta data or a specific message type + specific data (json?) in body?

Reply:
Done with CSS instead